Modules/SQLDatabase/2.0/Tests/parameters.json

```diff
@@ -7,6 +7,12 @@
         },
         "serverName": {
             "value": "bestesqlserver0"
```

Comment:
Do we need to rename the server name to be generic and based on standard naming convention?

Reply:
I already did, take a look at the updated code, not sure why is not showing up here, you can see next to the file name that says: Outdated.